sycl/include/CL/sycl/accessor.hpp

```diff
@@ -941,12 +928,11 @@ class accessor
       // arguments to avoid the need in adding utility functions for
       // dummy/default initialization of range<Dimensions> and
       // id<Dimension> fields.
-      : impl(nullptr, Range, bufferRef.MemRange,
-             &commandGroupHandlerRef, Offset) {}
+      : impl((_ValueType *)nullptr, Range, bufferRef.MemRange, Offset) {}
 #else   // !__SYCL_DEVICE_ONLY__
       : impl(std::make_shared<_ImplT>(
-                 (dataT *)detail::getSyclObjImpl(bufferRef)->BufPtr, Range,
-                 bufferRef.MemRange, &commandGroupHandlerRef, Offset)) {
+                 (_ValueType *)detail::getSyclObjImpl(bufferRef)->BufPtr,
```

Comment:
Idem

Reply:
I fixed the patch - used reinterpret_cast instead of C-style conversion and removed C-style conversion before nullptr. Thank you.